CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ To test the plugin, or to contribute to it, you can clone this repository and bu
 First, you need a WordPress Environment to run the plugin on. The quickest way to get up and running is to use the provided docker setup. Just install [docker](https://www.docker.com/) on your machine and run `./bin/setup-local-env.sh`.
 
 The WordPress installation should be available at `http://localhost:8888` (username: `admin`, password: `password`).
-Inside the "docker" directory, you can use any docker command to interact with your containers.
+Inside the "docker" directory, you can use any docker command to interact with your containers. If this port is in use, you can override it in your `docker-compose.override.yml` file.
```

Comment:
Thanks for this Gary 💯 

The Cypress `baseUrl` also needs to be overwritten for this to work I believe

Adding the following to your `cypress.env.json` file should do the trick I believe, though I didn't get this far in my local attempts to get an alternate port working:
```json
{
  "CYPRESS_BASE_URL": "http://localhost:8888"
}
```

See https://docs.cypress.io/guides/guides/environment-variables.html#Overriding-Configuration

Reply:
I was just writing the docs for this. 🙂

Reply:
So, it turns out that `baseUrl` can't be overridden from `cypress.env.json`. Also, it seems that file is probably going to be deprecated at some point. Working on an automated way of doing it.

Reply:
Just noting that `package.json` file has `config` field available:
https://docs.npmjs.com/files/package.json#config

I think Cypress is able to consume it. Let me find the related doc.

Reply:
It might be possible to set it up using `Cypress.config()`:

https://docs.cypress.io/guides/references/configuration.html#Environment-Variables

Reply:
I mean, I could do that. Or I could write a silly little Cypress plugin. 😉